### DISCUSSION TITLE
Manual correction of beatmap difficulty level

### DISCUSSION THREAD

**Discussion by guanjiao23456** [ppy/osu#27782] [Ideas] [answered]
The difficulty level of beatmaps in osu is automatically calculated by a computer, which in most cases is more accurate than manually determining the difficulty level, but sometimes beatmaps that the computer doesn't consider to be that difficult feel difficult to play.
I don't know what the computer's algorithm for calculating the difficulty level is, but it seems to me that the computer ignores the placement of notes in the beatmap. Maybe the computer thinks that notes that are close together are less difficult to play, but if they are arranged in a complex way (or in a way that is rarely seen that makes people uncomfortable), the computer will think it's easy, but a person might think it's hard!
Also consider how fast the center circle of the beatmap shrinks, etc.
In summary, it is recommended that the difficulty level of some beatmaps be manually corrected

Computerized translations may not be accurate, so please understand.

Translated with DeepL.com (free version)

**Comment by Joehuu** [comment, marked as answer]
I think this idea is similar to https://github.com/ppy/osu/discussions/26891.

**Reply by Rekunan** [reply]
Correct, and like I said over there, there are future plans for the pp system to account for more things, it's only a matter of time.

If you want to view the code, it's all in these 2 folders:
https://github.com/ppy/osu/tree/master/osu.Game.Rulesets.Osu/Difficulty
https://github.com/ppy/osu/tree/master/osu.Game/Rulesets/Difficulty
